index.html

```diff
@@ -10367,26 +10488,37 @@ <h4 id="Beast Shape - Large Flying Animal">Beast Shape - Large Flying Animal</h4
             <tr>
                <td>Core
                </td>
+               <td>Razor Talons
+               </td>
+               <td>Make a melee attack against a creature, dealing Medium+Str damage.
+               </td>
+               <td>E
+               </td>
+               <td>2
+               </td>
+            </tr>
+            <tr>
+               <td>1
+               </td>
                <td>Pick Up and Drop
                </td>
                <td>Sink your talons into an enemy. Make a regular melee attack. On a
-                  successful hit deal Medium+DEX damage. Hold onto the target and fly 20ft up into
-                  the air after which the target is dropped dealing 20ft fall damage (Heavy). 2
-                  FORT to resistance being lifted into the air
+                  successful hit deal Light+STR damage. You may then chose to hold onto the target 
+                  and fly 20ft up into the air after which the target is dropped dealing 20ft fall damage (Heavy). 
```

Comment:
It isn't necessarily Heavy anymore, unless you want to make it specifically Heavy and ignore the normal fall rules.